wgpu-hal/Cargo.toml

```diff
@@ -61,6 +70,8 @@ glow = { git = "https://github.com/grovesNL/glow/", rev = "c8a011fcd57a5c68cc917
 # backend: Dx12
 bit-set = { version = "0.5", optional = true }
 range-alloc = { version = "0.1", optional = true }
+winapi = { version = "0.3", features = ["libloaderapi", "windef", "winuser", "dcomp"] }
+native = { package = "d3d12", version = "0.5.0", features = ["libloading"], optional = true }
```

Comment:
Why do these need to be added to general dependencies? We only need them on windows.

Reply:
I guess I forgot to move these back, I was testing the documentation and tried to see if I could get these to build on Linux where docs.rs generates docs